apps/labs/posts/automated-accessibility-testing-with-bokeh.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Automated Accessibility Testing with Bokeh'
+authors: [chiemezuo-akujobi]
+published: September 10, 2024
+description: 'Data is only useful to the extent that people can access it.
+  Let me walk you through one of several steps toward making Bokeh data
+  visualizations more accessible.'
+category: [Access-centered, Beyond PyData, Internship]
+featuredImage:
+  src: /posts/hello-world-post/featured.png
+  alt: 'Excellent alt-text describing the featured image which appears as a
+    small image in our main blog page. This image needs to be 391x174 pixels
+    in size.'
+hero:
+  imageSrc: /posts/hello-world-post/hero.jpeg
+  imageAlt: 'Excellent alt-text describing the hero image which appears as a
+    large image at the top of your post page. This image needs to be 1440x696
+    pixels in size'
+---
+
+My name is Chiemezuo, and I got to do some amazing work as a Quansight Labs
+intern this summer. My project was on developing Automated Accessibility Testing
+for the [Bokeh](https://bokeh.org/) project, and I'm here to tell you all about
+it. The best part about this story is that neither my mentor nor I had
+previously worked with the codebase prior to this internship.
+
+## What is Bokeh?
+
+Bokeh is a Python library for creating interactive visualizations for modern web
+browsers. It lets you build beautiful graphics, from plots to dashboards. An
+important part of Bokeh is mentioned in the definition, and it's the word
+"interactive". Bokeh visualizations have a built-in interaction layer that makes
```

Comment:
You shine a spotlight on the word interactive, so you should explain right away why that's important, but you don't.

```suggestion
browsers. It lets you build beautiful graphics, from plots to dashboards. Bokeh visualizations have a built-in interaction layer that makes
```